pycalphad/model.py

```diff
@@ -430,9 +431,12 @@ def degree_of_ordering(self):
     #pylint: disable=C0103
     # These are standard abbreviations from Thermo-Calc for these quantities
     energy = GM = property(lambda self: self.ast)
+    energy_implementation_units = GM_implementation_units = 'J / mol'
+    energy_display_units = GM_display_units = 'kJ / mol'
     formulaenergy = G = property(lambda self: self.ast * self._site_ratio_normalization)
```

Comment:
These specific lines will be deleted because it's duplicative with what's in `pycalphad/property_framework/units.py`, but your point still stands.

`energy_{implementation,display}_units` needs to have consistent units with `Model.energy`, which is equal to `Model.GM`.

Eventually we can use the units support to move between formula units and per-mole-atom units and deprecate the `M` suffix defined directly on `Model`, but we're not all the way there yet.